.gitignore

```diff
@@ -24,3 +24,4 @@ bin/
 .pytest_cache
 pytest_session.txt
 python/*.dist-info/
+*~
```

Comment:
I thought these were automatically suppressed by emacs (although maybe only if the file is added?).  No objection to including it if you ran into it.

Reply:
These are created by emacs and `git status` complains about them.  Because of all the pipelines I've worked on I have tons of these here, and it finally broke me.